mlflow/genai/judges/utils.py

```diff
@@ -33,14 +35,74 @@ def format_prompt(prompt: str, **values) -> str:
     return prompt
 
 
+def add_output_format_instructions(prompt: str, output_fields: list[Any] | None = None) -> str:
```

Comment:
Can we also update the case where we invoke `__call__(trace=...)` to use this utility?